coffee_lib/src/plugin.rs

```diff
@@ -26,27 +26,21 @@ pub enum PluginLang {
 }
 
 impl PluginLang {
-    pub async fn default_install(
-        &self,
-        path: &str,
-        name: &str,
-        verbose: bool,
-    ) -> Result<String, CoffeeError> {
+    pub async fn default_install(&self, root_path: &str, verbose: bool) -> Result<(), CoffeeError> {
         match self {
             PluginLang::PyPip => {
                 /* 1. RUN PIP install or poetry install
                  * 2. return the path of the main file */
                 let script = "pip3 install -r requirements.txt";
-                sh!(path, script, verbose);
-                let main_file = format!("{path}/{name}.py");
-                Ok(main_file)
+                sh!(root_path, script, verbose);
+                Ok(())
             }
             PluginLang::PyPoetry => {
                 let script = "pip3 install poetry \
                               poetry export -f requirements.txt --output requirements.txt \
                               pip3 install -r requirements.txt";
-                sh!(path, script, verbose);
-                Ok(format!("{path}/{name}.py"))
+                sh!(root_path, script, verbose);
+                Ok(())
             }
             PluginLang::Go => Err(error!(
                 "golang is not supported as default language, please us the coffee.yml manifest"
```

Comment:
No this is wrong why we are removing the logic here?

why there is the need to remove the path as return type?